src/sc_scda.h

```diff
@@ -304,7 +304,7 @@ sc_scda_fopen_options_t; /**< type for \ref sc_scda_fopen_options */
  * [preprint](https://doi.org/10.48550/arXiv.2307.06789).
  *
  * This function returns NULL on MPI I/O errors.
- * Without MPI I/O the function may abort on file system dependent errors.
+ * Without MPI the function may abort on file system dependent errors.
```

Comment:
A question not for this PR, just to keep in mind: will we have non-abort error checks without MPI?